cellxgene_schema_cli/cellxgene_schema/utils.py

```diff
@@ -202,3 +202,36 @@ def is_ontological_descendant_of(onto: OntologyParser, term: str, target: str, i
 @lru_cache()
 def get_descendants(onto: OntologyParser, term: str, include_self: bool = True) -> List[str]:
     return onto.get_term_descendants(term, include_self=True)
+
+def check_non_csr_matrices(adata: ad.AnnData):
+    """
+    Check X, raw.X and layers matrices for having more than 50% zeros and not being csr_matrix
+
+    If found, convert to csr_matrix
+    """
+
+    def get_sparsity(matrix, format):
+        if format in SPARSE_MATRIX_TYPES:
+            nnz = matrix.nnz
+        else:
+            nnz = np.count_nonzero(matrix)
+        sparsity = 1 - nnz / np.prod(matrix.shape)
+        return sparsity
+
+    format = get_matrix_format(adata, adata.X)
```

Comment:
[get_matrix_format()](https://github.com/chanzuckerberg/single-cell-curation/blob/ed19edac8de15896d9eb06897cd372e4c48c7737/cellxgene_schema_cli/cellxgene_schema/utils.py#L75) looks like it takes a single argument. So, it should be `get_matrix_format(adata.X)`. And same with line 225